torch/testing/_internal/common_modules.py

```diff
@@ -1143,6 +1143,7 @@ def module_inputs_torch_nn_LSTM(module_info, device, dtype, requires_grad, train
                module_inputs_func=partial(module_inputs_torch_nn_ConvNd, N=2, lazy=False, transposed=True),
                gradcheck_nondet_tol=GRADCHECK_NONDET_TOL,
                module_memformat_affects_out=True,
+               dtypes=floating_and_complex_types_and(torch.chalf),
```

Comment:
maybe just add `dtypesIfCUDA=floating_and_complex_types()` so that we don't have to add skips for chalf cpu

Reply:
ModuleInfo doesn't have a notion of device specific dtypes.

https://github.com/pytorch/pytorch/blob/6faa6c68e8b76fb68f3a2b2783685102d0e87c00/torch/testing/_internal/common_modules.py#L165-L177